clients/pkg/promtail/targets/docker/target.go

```diff
@@ -95,7 +95,12 @@ func (t *Target) processLoop(ctx context.Context) {
 		Timestamps: true,
 		Since:      strconv.FormatInt(t.since, 10),
 	}
-
+	inspectInfo, err := t.client.ContainerInspect(ctx, t.containerName)
```

Comment:
sorry, i dont know how to stub this call in tests in this moment, could you suggest please? or somebody? i was thinking about delete 100 row return for dont failing cycle if inspectInfo has err, just log it with warning
i can commit suggest and merge in main
thank u!

Reply:
@youngpabl0 I've created the tests. It would be nice if you could merge https://github.com/youngpabl0/loki/pull/1. Please not that my branch includes newer commits.
https://github.com/youngpabl0/loki/pull/1/files#diff-4354d1db0b5df6f58022a408048b806a0992c0bd352c3ed062e0427d218f519c

It would be really nice if you could add a test with `Tty: true`. Ideally in a table driven test.The tough bit is getting some sample logs.